e2e/docker/compose.yaml.tmpl

```diff
@@ -34,20 +34,15 @@ services:
 {{end}}
 
 {{- range .Anvils }}
-  # Initialises geth files and folder from provided genesis file.
   {{ .Chain.Name }}:
     labels:
       e2e: true
     container_name: {{ .Chain.Name }}
     platform: linux/amd64
```

Comment:
Do we still need the platform here?

Reply:
I'm not sure

Reply:
We've only build for [amd64](https://hub.docker.com/repository/docker/omniops/forkproxy/tags) so yes